src/third-parties/google-analytics/data.json

```diff
@@ -15,7 +15,7 @@
       "key": "gtag"
     },
     {
-      "code": "window[{{l}}]=window[{{l}}]||[];window[{{l}}].push({'js':new Date()});window[{{l}}].push({'config':{{id}}})",
+      "code": "window[{{l}}]=window[{{l}}]||[];function gtag(){window[{{l}}].push(arguments);};gtag('js',new Date());gtag('config','{{id}}')",
```

Comment:
just pushing init code in the dataLayer doesn't work. similar to the original implementation we are using a gtag function. We are just not making it global by NOT assigning it to the windows object.

Reply:
`gtag` was removed because having multiple gtm/ga would override the last `gtag`

Reply:
it's not on windows. we can also use any function name we like, ie:

window['gtag-{{l}}] = function(){}

Reply:
well... let me use the dynamic approach...uno momento